pkg/spdk/disk.go

```diff
@@ -283,7 +293,7 @@ func addBlockDevice(spdkClient *SPDKClient, diskName, diskPath string, blockSize
 func lvstoreToDisk(spdkClient *SPDKClient, diskPath, lvstoreName, lvstoreUUID string) (*spdkrpc.Disk, error) {
 	lvstores, err := spdkClient.BdevLvolGetLvstore(lvstoreName, lvstoreUUID)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to get lvstore with name %v and UUID %v", lvstoreName, lvstoreUUID)
+		return nil, errors.Wrapf(err, "cannot find lvstore with name %v and UUID %v", lvstoreName, lvstoreUUID)
```

Comment:
No need to update this log. This is an API invocation failure rather than not finding an entry in a valid list/map

Reply:
Reverted. Thank you.